### DISCUSSION TITLE
Need help with RSS feed and images in posts

### DISCUSSION THREAD

**Discussion by cx48** [welpo/tabi#363] [Information]
In config.toml I have feed_icon set to true. Do I need to do something else for RSS icon and link? I want to enable RSS

Another issue I'm facing is with Images. I have added all the images in static/img in my posts I'm trying to add images using markdown which is `![Image alt](/static/img/file)` How can I add images correctly? This doesn't seem to work for me

**Comment by welpo** [comment]
Hello!

To enable RSS, you need to configure Zola and tabi. The only tabi setting needed is `feed_icon`, which you've already set.

On the Zola side, in `config.toml` (before the `extra` section), set:

```toml
generate_feeds = true
```

For Zola 0.19.x. If you're using Zola 0.18.x, the setting is singular:

```toml
generate_feed = true
```

---

For the images, try `![Image alt](/img/file)`.

The `static` directory is not created on build, so the parent directory is `/`, not `/static/`.

Hope that helps!

**Reply by cx48** [reply]
`generate_feeds = true` before `[extra]` section and `/img/file` for adding images worked out flawlessly!

Thanks!

**Comment by welpo** [comment]
By the way, you might find this useful: https://www.getzola.org/documentation/content/overview/#asset-colocation

Particularly if you're going to use many images/content per post.

**Comment by cx48** [comment]
1. Do you know any solution to this? Zola preset has been selected before build. It keeps failing. It's more of a Vercel, Zola issue.
```
Warning: Failed to fetch one or more git submodules
Cloning completed: 457.891ms
Running "vercel build"
Vercel CLI 35.2.3
sh: line 1: zola: command not found
Error: Command "zola build" exited with 127
```

2. Are there any other signs in tabi?
```
{{ admonition(type="warning", title="DEPRECATION WARNING", text="Zola v0.19.0 and later can do this natively. Set `bottom_footnotes = true` in your config's `[markdown]` section.") }}
```
I found only a warning sign from your posts. Red for danger and a green or blue for a note would be cool.

**Reply by welpo** [reply]
> 2\. Are there any other signs in tabi?

Yes: https://welpo.github.io/tabi/blog/shortcodes/#admonitions

> Do you know any solution to this? Zola preset has been selected before build. It keeps failing. It's more of a Vercel, Zola issue.

Sorry, I don't use Vercel. Did you read this? https://www.getzola.org/documentation/deployment/vercel/

**Reply by cx48** [reply]
> Yes: https://welpo.github.io/tabi/blog/shortcodes/#admonitions

Thank you!
> Sorry, I don't use Vercel. Did you read this? https://www.getzola.org/documentation/deployment/vercel/

It's okay! I will check it out, thank you!
